scripts/deploy/phoenix.dataverse.org/post

```diff
@@ -1,6 +1,6 @@
 #/bin/sh
 cd scripts/api
-./setup-all.sh | tee /tmp/setup-all.sh.out
+./setup-all.sh --insecure | tee /tmp/setup-all.sh.out
```

Comment:
Right. We need http://phoenix.dataverse.org to be installed in "insecure" mode so that we can run the integration tests at https://build.hmdc.harvard.edu:8443/job/phoenix.dataverse.org-apitest-develop/ . Thanks for adding this.